src/App.js

```diff
@@ -12,12 +12,30 @@ function App() {
   const [facturas, setFacturas] = useState([]);
   /* const { datos: facturasAPI } = useFetch(`${process.env.REACT_APP_API_URL}`); */
   const { datos: facturasAPI } = useFetch(urlFacturas);
+  const [totalBase, setTotalBase] = useState(0);
+  const [totalIva, setTotalIva] = useState(0);
+  const [totalTotal, setTotalTotal] = useState(0);
+
+  useEffect(() => {
+    if (facturas.length > 0) {
```

Comment:
No es necesario todo este if si a los reduce le pasáis un 0 como valor inicial del acumulador.